runtime/src/utxo.rs

```diff
@@ -56,84 +52,44 @@ pub struct TransactionOutput {
 	/// owner must provide a proof by hashing whole `TransactionOutput` and
 	/// signing it with a corresponding private key.
     pub pubkey: H256,
-
-    /// Unique (potentially random) value used to distinguish this
-	/// particular output from others addressed to the same public
-	/// key with the same value. Prevents potential replay attacks.
-    pub salt: u64,
-}
-
-/// A UTXO can be locked indefinitely or until a certain block height
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
-#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, Hash)]
-pub enum LockStatus<BlockNumber> {
-    Locked,
-    LockedUntil(BlockNumber),
 }
 
 decl_storage! {
     trait Store for Module<T: Trait> as Utxo {
         /// All valid unspent transaction outputs are stored in this map.
         /// Initial set of UTXO is populated from the list stored in genesis.
-        UnspentOutputs build(|config: &GenesisConfig| {
-            config.initial_utxo
+        UtxoStore build(|config: &GenesisConfig| {
+            config.genesis_utxo
                 .iter()
                 .cloned()
                 .map(|u| (BlakeTwo256::hash_of(&u), u))
```

Comment:
Yeah I sort of left this alone bc making this change meant I'd have to recalculate all the inputs for the demo (super manual) and rewrite the tests. But I should probably still do this, otherwise users will get confused... I'll turn this into a future issue.